src/qrl/core/TransactionInfo.py

```diff
@@ -8,8 +8,17 @@
 
 
 class TransactionInfo:
-
+    """
+    TransactionInfo is a container used by TransactionPool to identify if a
+    particular TX is stale, so it includes information like when you received
```

Comment:
1. It is not just to identify if a particular TX is stale. Rather the timestamp is used by the blockexplorer.
2. Good to mention unconfirmed Transaction instead of TX.
3. Replace TX by Transaction. Please don't use shortcuts while writing documentation.